CMakeLists.txt

```diff
@@ -270,6 +270,13 @@ cxx_feature_check(STEADY_CLOCK)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 
+if (BENCHMARK_LIBPFM_LIB_DIR AND BENCHMARK_LIBPFM_INCLUDE_DIR)
```

Comment:
should this be using `find_package` instead?

Reply:
I see, so if libpfm is available, then the build is auto-enabling counters support - or should there still be a master switch?